src/main/java/Warehouse/WarehouseManager/product/Product.java

```diff
@@ -2,24 +2,40 @@
 
 import Warehouse.WarehouseManager.enums.ProductSize;
 import jakarta.persistence.*;
-import lombok.AllArgsConstructor;
-import lombok.Data;
-import lombok.NoArgsConstructor;
+import lombok.*;
 import org.hibernate.engine.jdbc.Size;
+import org.springframework.stereotype.Service;
+
+import java.util.*;
 
 @AllArgsConstructor
 @NoArgsConstructor
-@Data
+@Getter
 @Entity
+
 public class Product {
     @Id
     @GeneratedValue(strategy = GenerationType.IDENTITY)
+    @Setter
     private Long id;
 
     @Column(unique = true, nullable = false, name = "product_name")
+    @Setter
     private String name;
 
     @Enumerated(EnumType.STRING)
     @Column(nullable = false, name = "product_size")
     private ProductSize size;
+
+    public ProductDto toProductDto(){
+        return new ProductDto(id,name,size);
+    }
+    public void setSize(ProductSize size) {
+        EnumSet<ProductSize> productSizeList =
+                EnumSet.of(ProductSize.SMALL,ProductSize.MEDIUM,ProductSize.BIG,ProductSize.PALETTE);
```

Comment:
Enum provides the `values()` method, which returns an array of all constants. I think that using it would make the code more flexible.